### DISCUSSION TITLE
Default chart settings

### DISCUSSION THREAD

**Discussion by Johnyryal** [alexjustesen/speedtest-tracker#1427] [Ideas]
Love the utility and it's been helpful for validating the service levels in our shared internet of our condo building. One request is to set default chart settings or keep the setting persistent with a cookie or something. I test once a night so having the default be 24hours means I always have to change them back to 1 month. Love the tool and the contributors here.

**Comment by p-jones** [comment]
Could not agree more. Please make this a persistent setting!

**Comment by alexjustesen** [comment]
Attaching to #738 as there is already an issue related to the charts and date range selections
